### PR TITLE
 Add New Adapter dgadsBidAdapter

### PR DESCRIPTION
**Pull request by r-sato**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'dgads',
  params: {
    location_id: '1',
    site_id: '1';
}
```

- contact email of the adapter’s maintainer:dgads-support@garage.co.jp
- [x] official adapter submission
- A link to a PR on the docs repo at prebid/prebid.github.io#714

### CODE REVIEW

File: modules/dgadsBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  // Banner
+  {
+    code: 'banner-div',
+    sizes: [[300, 250]],
+    bids: [{
+      bidder: 'dgads',
+      mediaTypes: 'banner',
+      params: {
+        location_id: '1',
+        site_id: '1';
```
Comment (jsnellbaker) [inline modules/dgadsBidAdapter.md]: Can you remove this semi-colon?

File: modules/dgadsBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    utils._each(ads, function(ad) {
+      bidResponse.requestId = ad.bidId;
+      bidResponse.bidderCode = BIDDER_CODE;
+      bidResponse.cpm = ad.cpm;
+      bidResponse.creativeId = ad.creativeId;
+      bidResponse.currency = 'JPY';
+      bidResponse.netRevenue = true;
+      bidResponse.ttl = ad.ttl;
+      bidResponse.referrer = utils.getTopWindowUrl();
+      if (ad.isNative == 1) {
```
Comment (jsnellbaker) [inline modules/dgadsBidAdapter.js]: Somewhere in here I would recommend to set the bid's mediaType to `native`.

When I was testing the native ad with the .md file's test params, the bid's `mediaType` was set to the default `banner`.

**Comment by r-sato** [comment]
Thank you for the review.
We fixed both and committed.

And regarding hello_world.html,
the image server was down, so we responded.
Please check again.

File: modules/dgadsBidAdapter.js (hunk 2 above)
Comment (r-sato) [reply modules/dgadsBidAdapter.js]: Thank you. We corresponded.

File: modules/dgadsBidAdapter.md (hunk 1 above)
Comment (r-sato) [reply modules/dgadsBidAdapter.md]: Thank you. We corresponded.

**Comment by jsnellbaker** [comment]
Hi @r-sato, thanks for making the changes.

I retested and found the banner creative was showing successfully.  However the native test ad wasn't rendering.  When I was looking at the browser console to debug the issue, I realized that the `pbjs.renderAd` function was being invoked - which is normally used for banner or video ads.  Native creatives use a different template to render themselves given the variety of creative assets that may/may not be used.  

The following page gives an outline of what the native creative code looks like: http://prebid.org/adops/setting-up-prebid-native-in-dfp.html  Can you take a look over this page?

**Comment by r-sato** [comment]
Hi @jsnellbaker, thank you for the check.

Our native ad unit is rendered successfully.
We have registered HTML and CSS with reference to http://prebid.org/adops/setting-up-prebid-native-in-dfp.html.

This is our test html for native.
[hello_world_native](https://ads-tr.bigmining.com/hello_world_native.zip)

Please let me know if anything differs.

**Comment by jsnellbaker** [comment]
@r-sato Thanks for making the updates and providing the sample test page.  Everything looks good now.

**Pull request merged**